src/backend/slack/slack.ts

```diff
@@ -21,7 +21,7 @@ export const slackNotify = (
     const formatertMelding: string = `*${displayName}, ${namespace}*\n ${req.body.melding}`;
 
     backend.logInfo(req, `Poster slack melding til #${kanal}: ${formatertMelding}`);
-    fetch('https://slack.com/api/chat.postMessage', {
+    /*fetch('https://slack.com/api/chat.postMessage', {
```

Comment:
Skal denne kodeblokka inn igjen på et senere tidspunkt, eller kan den fjernes helt? :)

Reply:
Den skal inn igjen, men ikke vits å spamme alerts før vi lanserer :) Kunne lagt inn noe logikk for når den skal kalle slack, men dette er enklest.